cluster-autoscaler/expander/price/price.go

```diff
@@ -72,13 +114,59 @@ nextoption:
 			continue
 		}
 
-		optionScore := totalNodePrice / totalPodPrice
-		glog.V(5).Infof("Price of %s expansion is %f - ratio %f", option.NodeGroup.Id(), totalNodePrice)
+		// How well the money is spent.
+		priceSubScore := (totalNodePrice + stabilizationPrice) / (totalPodPrice + stabilizationPrice)
+		// How well the node matches generic cluster needs
+		nodeUnfitness := p.nodeUnfitness(preferredNode, nodeInfo.Node())
+
+		// TODO: normalize node count against preferred node.
```

Comment:
Any reason not to do that now? It seems simple enough. Or is it large enough to make it more readable by putting it in separate pr?